src/clue/clue.sass

```diff
@@ -1,2 +1,19 @@
 // application-specific styling goes here
-  
\ No newline at end of file
+
```

Comment:
Are we not using the color "themes" from the vars.sass files anymore?

Reply:
🤦‍♂ Constants? What constants?